rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -1330,6 +1330,18 @@ error_code sys_fs_fcntl(ppu_thread& ppu, u32 fd, u32 op, vm::ptr<void> _arg, u32
 
 	switch (op)
 	{
+	case 0x80000004: // Unknown
+	{
+		if (_size > 4)
+		{
+			return CELL_EINVAL;
+		}
+
+		const auto arg = vm::static_ptr_cast<u32>(_arg);
+		*arg = 0;
```

Comment:
What if _arg is null in this function?

Reply:
Not like we handle nulls for any other case in this method..

Reply:
But can you double check?

Reply:
the fcntl code is a right PITA to figure out the location of, so i don't know how it would be handled, however this fcntl has only ever been used by vsh via lle library, so its very unlikely to ever happen

Reply:
This pointer comes from cellFs LLE hence it is always valid.